x-pack/plugins/timelines/common/types/timeline/index.ts

```diff
@@ -465,10 +465,6 @@ export enum TimelineTabs {
   eql = 'eql',
 }
 
-export type CreateFieldComponentType = React.FC<{
```

Comment:
all field browser types moved to its own place in the common types [x-pack/plugins/timelines/common/types/fields_browser/index.ts](https://github.com/elastic/kibana/pull/126105/files#diff-cb0b9396b00a448cf0e406b4ef3d39f0336e3dcb3f63e12ff4e9ab7b2ac1957b)
The reason is the common tGrid type now has a new `fieldBrowserOptions` prop, which needs to be typed, they can not be imported from outside /common directory.